e2e/specs/drug-orders.spec.ts

```diff
@@ -1,228 +1,285 @@
 import { expect } from '@playwright/test';
 import { type Visit } from '@openmrs/esm-framework';
-import { generateRandomPatient, deletePatient, type Patient, startVisit, endVisit } from '../commands';
+import {
+  generateRandomPatient,
+  deletePatient,
+  generateRandomDrugOrder,
+  deleteDrugOrder,
+  createEncounter,
+  deleteEncounter,
+  getProvider,
+  type Patient,
+  startVisit,
+  endVisit,
+} from '../commands';
+import { type Encounter, type Provider } from '../types';
+import { type Order } from '@openmrs/esm-patient-common-lib';
 import { test } from '../core';
-import { MedicationsPage } from '../pages';
+import { OrdersPage } from '../pages';
 
 let patient: Patient;
 let visit: Visit;
+let drugOrder: Order;
+let encounter: Encounter;
+let orderer: Provider;
+let url: String;
```

Comment:
```suggestion
const url = process.env.E2E_BASE_URL;
```

It looks like the value doesn't changes throughout the test execution.